messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/BridgeDefinition.java

```diff
@@ -185,7 +185,8 @@
             PRODUCER_WINDOW_SIZE,
             CommonAttributes.BRIDGE_CONFIRMATION_WINDOW_SIZE,
             USER, PASSWORD, CREDENTIAL_REFERENCE,
-            CONNECTOR_REFS, DISCOVERY_GROUP_NAME
+            CONNECTOR_REFS, DISCOVERY_GROUP_NAME,
+            CommonAttributes.CALL_TIMEOUT
```

Comment:
I'm wondering if the bridge should have its own attribute definition

Reply:
OK, created it's own attr definition.